app/models/field_of_operation.rb

```diff
@@ -0,0 +1,5 @@
+class FieldOfOperation < ContentItem
+  def fatality_notices
```

Comment:
I think we can combine the commit [Add field_of_operation model test](https://github.com/alphagov/frontend/pull/4742/commits/633bb78c3cfe6ee129e82c1d552c78ed2bedc23d) into this commit [Add FieldOfOperation model](https://github.com/alphagov/frontend/pull/4742/commits/c5bea5832af58c38e8e2ffb277602bd77a7bc866)